podcasts/EpisodeDetailViewController+ShowNotes.swift

```diff
@@ -35,13 +35,10 @@ extension EpisodeDetailViewController: WKNavigationDelegate, SFSafariViewControl
         hideErrorMessage(hide: true)
 
         if FeatureFlag.newShowNotesEndpoint.enabled {
-            let podcastUUID = episode.parentIdentifier()
-            let episodeUUID = episode.uuid
             Task { [weak self] in
-                if let showNotes = await self?.episode.loadMetadata()?.showNotes {
-                    self?.downloadingShowNotes = false
-                    self?.showNotesDidLoad(showNotes: showNotes)
-                }
+                let showNotes = await self?.episode.loadMetadata()?.showNotes
```

Comment:
Just a small fix that I thought about including here. When the show notes were not retrieved the error message wouldn't appear, this fixes it.